Create rules page (#2)
osuTitanic/wiki: wiki/Rules/en.md

# Rules

These are the rules that help keep the game and community fun and fair.

# Community Rules

> 1. **Each user may have only *one* account EVER!** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you cannot access your email or you have forgot your password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).

Reviewer: ```suggestion
1. **Each user may only *one* account EVER!** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you cannot access your email or you have forgot your password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```

> 1. **Each user may have only *one* account EVER!** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you cannot access your email or you have forgot your password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).

Reviewer: ```suggestion
1. **Each user may have only *one* account EVER!** The only exception to this rule is making bot accounts or other projects, but you must let staff know so we can approve it. If you cannot access your email or you have forgot your password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```

2. **Play fair.** Using any third party tools that gives you an unfair advantage over people will result in a restriction. This includes programs like macros, cheat engine, aim-assist programs, timewarp, and so on. If a program is doing something to help you play that you should be doing then it is not okay! If you see some suspicious activity from the user, please open up a ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc) server.
3. **Be kind to people.** Any harrassment, racism, or antagonism has no place here in Titanic! This is just a server about clicking circles, catching fruit, bashing drums, and breaking keyboards whilst pretending to be cookiezi and playing old maps, not to be assholes to each other.

> 4. **We are an all-ages community.** This means that any 18+/NSFW content such as drug use or topics of sexual nature are not welcome here. 

Reviewer 2: I would not say that we are an "all-ages" community. The minimum age is 13+ just to be safe here.

Reviewer: yeah i actually agree.

Author: I guess making it something on the lines of, "No NSFW/18+ Content allowed" instead of all-ages.

# Rules

These are the rules that help keep the game and community fun and fair.

# Community Rules

> 1. **Each user may have only one account EVER!** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you have lost your email or password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).

Reviewer: ```suggestion
1. **Each user may have only *ONE* account ever.** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you cannot access your email or you forgot the password, please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```

Reviewer: oh apologize, put the caps on "EVER"

2. **Play fair.** Using any third party tools that give you an unfair advantage over people will result in a restriction. This includes programs like macros, cheat engine, aim-assist programs, timewarp, and so on. If a program is doing something to help you play that you should be doing then it is not okay! If you catch anybody cheating please open up a ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc) server.

> 3. **Be kind to people.** Any harrassment, racism, or antagonism has no place here in Titanic! This is just a server about clicking circles, catching fruit, bashing drums, and breaking keyboards whilst pretending to be cookiezi and playing old maps, not to be assholes to each other. 

Reviewer: ```suggestion
3. **Be kind to people.** Any harrassment, racism, or antagonism has no place here in Titanic! This server is about clicking circles, catching fruit, bashing drums, and breaking keyboards whilst pretending to be Cookiezi and playing old maps, not to be assholes to each other. 
```
